### PR TITLE
Share To extension default parameter support

### PR DESCRIPTION
**Pull request by buracules**
- Add default title, description and image support for share to extension.
- Twitter is not working right now probably need to access via API

### How was this tested?
- [x] I tested this manually
- [ ] I added automated tests
- [ ] Existing automated tests are enough to cover my changes

### Screenshots and screen recordings
|Before|After|
|---|---|
|IMG_4938|IMG_4937|
|IMG_4939|IMG_4940|

## Issues addressed
* #3594

### CODE REVIEW

**Review by yozuysal — COMMENTED** [review]
I think this overall looks OK, one question about Kanna dependency.

Another thing is we will need to consolidate this with SocialInfoUpdater somehow. Can we have a setup where there are two fetch routes but we process the info we fetched in the same way in ShareTo and inside the app. See SocialInfoUpdater.swift for how it handled instagram and twitter.

File: Extensions/ShareTo/ShareToViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                        self.loading = false
+                    }
+                }
+
+            }
+        }.resume()
+    }
+
+    private func processsMetatag(_ xmlElement: XMLElement) {
+        switch xmlElement["property"] {
```
Comment (yozuysal) [inline Extensions/ShareTo/ShareToViewModel.swift:64]: should we put the properties here in an enum?

File: Neeva.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 2]
```diff
@@ -37,6 +37,15 @@
           "version": "1.4.3"
         }
       },
+      {
+        "package": "Kanna",
```
[the remark this answers is not in the document]
Comment (yozuysal) [reply Neeva.xcworkspace/xcshareddata/swiftpm/Package.resolved:41]: ping on this question

**Comment by buracules** [comment]
@yozuysal Removed Kanna from Client in my last commit probably.

File: Neeva.xcworkspace/xcshareddata/swiftpm/Package.resolved (hunk 2 above)
[the remark this answers is not in the document]
Comment (buracules) [reply Neeva.xcworkspace/xcshareddata/swiftpm/Package.resolved:41]: Removed Kanna from Client in my last commit

**Review by yozuysal — APPROVED** [review]
lgtm % the enum nit

File: Extensions/ShareTo/ShareToView.swift [hunk 3]
```diff
@@ -3,70 +3,17 @@
...
-                    VStack(spacing: 0) {
-                        NavigationLink(
-                            destination: AddToSpaceView(item: item, onDismiss: onDismiss)
-                        ) {
-                            ShareToAction(
-                                name: "Save to Spaces",
-                                icon: Symbol(decorative: .bookmark, size: 18))
-                        }
-                        Divider()
-                        OpenInNeevaView(item: item, incognito: false)
```
[the remark this answers is not in the document]
Comment (buracules) [reply Extensions/ShareTo/ShareToView.swift:35]: I am going to send another pr to fix this

[the remark this answers is not in the document]
Comment (jedireza) [reply Extensions/ShareTo/ShareToView.swift:35]: https://github.com/neevaco/neeva-ios/pull/3617

**Pull request merged**